lib/graphql/schema/warden.rb

```diff
@@ -106,6 +106,12 @@ def get_field(parent_type, field_name)
         @visible_parent_fields[parent_type][field_name]
       end
 
+      # @return [GraphQL::Argument, nil] The argument named `argument_name` on `parent_type`, if it exists and is visible
+      def get_argument(parent_type, argument_name)
```

Comment:
I believe the intent of this method was to be cached which it is not right now?

Reply:
`visible_argument?(argument)` is cached through the `visible?` method on types. So I believe this is ok to leave non-cached?

I can make it more similar to the above methods though and cache it further.

Reply:
I'll defer to @rmosolgo's judgement here. I did note that we are now calling `get_argument` twice on the same argument in some scenarios.

Reply:
Yeah, the call to the application's implementation would be cached, I don't know whether using `read_through { ... }` here is worth it or not, no preference without actually seeing a benchmark.